lighthouse-core/scripts/roll-to-devtools.sh

```diff
@@ -29,26 +30,36 @@ if [[ ! -d "$frontend_dir" || ! -a "$frontend_dir/Runtime.js" ]]; then
   echo "    $frontend_dir"
   exit 1
 else
-  echo -e "\033[96m ✓\033[39m Chromium folder in place."
+  echo -e "$check Chromium folder in place."
 fi
 
-report_dir="lighthouse-core/report/html"
 fe_lh_dir="$frontend_dir/audits/lighthouse"
 
 lh_bg_js="dist/lighthouse-dt-bundle.js"
-lh_worker_dir="$frontend_dir/audits_worker/lighthouse"
+fe_worker_dir="$frontend_dir/audits_worker/lighthouse"
 
 # copy lighthouse-dt-bundle (potentially stale)
-cp -pPR "$lh_bg_js" "$lh_worker_dir/lighthouse-dt-bundle.js"
-echo -e "\033[96m ✓\033[39m (Potentially stale) lighthouse-dt-bundle copied."
+cp -pPR "$lh_bg_js" "$fe_worker_dir/lighthouse-dt-bundle.js"
+echo -e "$check (Potentially stale) lighthouse-dt-bundle copied."
 
 # copy report generator + cached resources into $fe_lh_dir
-# use dir/* format to copy over all files in dt-report-resources directly to $fe_lh_dir 
+# use dir/* format to copy over all files in dt-report-resources directly to $fe_lh_dir
 # dir/ format behavior changes based on if their exists a folder named dir, which can get weird
-cp -r dist/dt-report-resources/* $fe_lh_dir
+cp -r dist/dt-report-resources/* "$fe_lh_dir"
+echo -e "$check Report resources copied."
+
+# copy locale JSON files
+lh_locales_dir="lighthouse-core/lib/i18n/locales/"
+fe_locales_dir="$frontend_dir/audits_worker/lighthouse/locales/"
+
+cp -r "$lh_locales_dir" "$fe_locales_dir"
```

Comment:
this copies the folder into the destination if the already exists.

![image](https://user-images.githubusercontent.com/4071474/64301653-be54fa80-cf34-11e9-83de-2af8b2c9d60b.png)

just do the glob cp, like above? `cp -r ./from/* ./to/`. could even drop the `-r`.